test/jdk/com/sun/management/HotSpotDiagnosticMXBean/DumpThreadsWithEliminatedLock.java

```diff
@@ -28,6 +28,7 @@
  *     an object that is scalar replaced
  * @requires !vm.debug & (vm.compMode != "Xcomp")
  * @requires (vm.opt.TieredStopAtLevel == null | vm.opt.TieredStopAtLevel == 4)
+ * @requires vm.jvmti
```

Comment:
This seems like a separate discussion as the minimal VM doesn't have the M&M support so something isn't right if somehow is testing a run-time image that contains jdk.management and only the minimal VM. I assume it's just a drive-by change here but I think part of a bigger discussion.

Reply:
> > This seems like a separate discussion as the minimal VM doesn't have the M&M support
> 
> Sure, it's maybe off topic. I now see that few tests handle this. I hit it as a minimal build showed me I had the wrong THROW macro, so in getting that right, I want to build and test such a VM. This test fails to load ManagementFactory with such a build (expected but confusing). It should skip.

I think you drop the change to the test as testing minimal VM is a bigger discussion. The make target for the compact profiles was removed in JDK 9 so would require running jlink to create a run-time image to test.